packages/Babylonian-UI.package/BPIcons.class/class/boxIconContents.st

```diff
@@ -1,48 +1,20 @@
-Zy8xOTk5LzAyLzIyLXJkZi1zeW50YXgtbnMjIj4KICA8cmRmOkRlc2NyaXB0aW9uIHJkZjph
-Ym91dD0iIgogICAgeG1sbnM6ZXhpZj0iaHR0cDovL25zLmFkb2JlLmNvbS9leGlmLzEuMC8i
-CiAgICB4bWxuczp0aWZmPSJodHRwOi8vbnMuYWRvYmUuY29tL3RpZmYvMS4wLyIKICAgIHht
-bG5zOnBob3Rvc2hvcD0iaHR0cDovL25zLmFkb2JlLmNvbS9waG90b3Nob3AvMS4wLyIKICAg
-IHhtbG5zOnhtcD0iaHR0cDovL25zLmFkb2JlLmNvbS94YXAvMS4wLyIKICAgIHhtbG5zOnht
-cE1NPSJodHRwOi8vbnMuYWRvYmUuY29tL3hhcC8xLjAvbW0vIgogICAgeG1sbnM6c3RFdnQ9
-Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC9zVHlwZS9SZXNvdXJjZUV2ZW50IyIKICAg
-ZXhpZjpQaXhlbFhEaW1lbnNpb249IjM2IgogICBleGlmOlBpeGVsWURpbWVuc2lvbj0iMzYi
-CiAgIGV4aWY6Q29sb3JTcGFjZT0iMSIKICAgdGlmZjpJbWFnZVdpZHRoPSIzNiIKICAgdGlm
-ZjpJbWFnZUxlbmd0aD0iMzYiCiAgIHRpZmY6UmVzb2x1dGlvblVuaXQ9IjIiCiAgIHRpZmY6
-WFJlc29sdXRpb249IjcyLjAiCiAgIHRpZmY6WVJlc29sdXRpb249IjcyLjAiCiAgIHBob3Rv
-c2hvcDpDb2xvck1vZGU9IjMiCiAgIHBob3Rvc2hvcDpJQ0NQcm9maWxlPSJzUkdCIElFQzYx
-OTY2LTIuMSIKICAgeG1wOk1vZGlmeURhdGU9IjIwMjEtMDItMTVUMTE6Mzc6MTMrMDE6MDAi
-CiAgIHhtcDpNZXRhZGF0YURhdGU9IjIwMjEtMDItMTVUMTE6Mzc6MTMrMDE6MDAiPgogICA8
-eG1wTU06SGlzdG9yeT4KICAgIDxyZGY6U2VxPgogICAgIDxyZGY6bGkKICAgICAgc3RFdnQ6
-YWN0aW9uPSJwcm9kdWNlZCIKICAgICAgc3RFdnQ6c29mdHdhcmVBZ2VudD0iQWZmaW5pdHkg
-RGVzaWduZXIgMS44LjUiCiAgICAgIHN0RXZ0OndoZW49IjIwMjEtMDItMTVUMTE6Mzc6MTMr
-MDE6MDAiLz4KICAgIDwvcmRmOlNlcT4KICAgPC94bXBNTTpIaXN0b3J5PgogIDwvcmRmOkRl
-c2NyaXB0aW9uPgogPC9yZGY6UkRGPgo8L3g6eG1wbWV0YT4KPD94cGFja2V0IGVuZD0iciI/
-PkTBwBgAAAGCaUNDUHNSR0IgSUVDNjE5NjYtMi4xAAAokXWRu0sDQRCHvySKopEIWlhYHL6q
-RHyAj8YiwReoRXKCUZvkzEPIxeMuImIr2AoKoo2vQv8CbQVrQVAUQewEa0UblXPOEyJiZpmd
-b3+7M+zOglfNabpV1gF6vmBGh8PKVHxaqXjES4AqmulNaJYxHhtSKWlvN3iceBVyapU+969V
-z6UsDTyVwgOaYRaER4THlgqGw5vC9Vo2MSd8LBw05YLC146edPnJ4YzLHw6bajQC3lphJfOL
-k79Yy5q6sLycFj23qP3cx3mJP5WfjElsEm/EIsowYRRGGSRCD530y9xDiC7aZUWJ/I7v/AkW
-JFeT2WAZk3kyZCkQFHVRqqckpkVPycix7PT/b1+tdHeXW90fhvIH235phYoN+Fy37fd92/48
-AN89nOWL+Qt70Pcq+npRa9mFwCqcnBe15BacrkHDnZEwE9+ST9ybTsPzEdTEoe4Sqmbcnv3s
-c3gL6op81QVs70CbnA/MfgGS3Wf6te970AAAAAlwSFlzAAALEwAACxMBAJqcGAAAAlJJREFU
-WIXVmD1PFFEUhp8lmECJodiEygYWC5KlwkpYTbbRChr/gUZ7NTQk/AAxIdpABXY0JlJg/IC/
-sIkmWsgPkC0IQkxgdyjm3Hi5HzN3Zu8KvMlN7sycj3cn7557zsAVQyVCjHFgVva7wM8IMUth
-WggkxtoF6v+TSBVYB7oOMmp1xabaTyJDwEvgyEh+CryXdWo8OwJeiG80VIAFYB/7TXwAJjTb
-mtwz7X4B80TQ7DSw50jwDWhm+DWB7w6/PUrqy6eTNvAUGAyIMSi2bXrU133gEFsnr4CR0CAa
-RoAVbH0dAvfynB9hvxVTJ2Xh0ldXcnrRMhx+ADMRyCjckZh6jlaWwwG2EBNgAxjrgciYxHDF
-Pshy1A3/GtfHwCLFasqw+BznxA4idAvYcvyiffJrSkVsXLVrS2IXJqTQwNZWAnwBphwxpoCv
-DvsWMJeTK4gQpDXlCbbGOsAbYFTWW7lnauQxdu3qiZDCTeA1cGbY/pGl3zsjrT++2hWFkMJt
-4KNhr68dsek5VyghSIX7kItHQxt4QNghGp2QwrLms1zAz5lroEAAHzqefSnEIBQV14pQP8l6
-Y5sPfmv7u/3hAvwbm8ycFqHP2n6T7Da1LJqkJ7/CpyzjOnDCxb/kNmlz5cOSZruUYVeTWHrs
-EwJ67AbuFnaF9OgoSkgdN64WtpFHRqEKrBHW5PsI3QCe4W7y1yg5RIaMQS5C0ccgHVnN1jaw
-ql2vYuskIeKgqGOIdCw2R+mOZ5+I7XMij9ImfPqKppOyqHNFPseYmATeyZq8TCJ9wTli4VXH
-fOdG1QAAAABJRU5ErkJggg=='
\ No newline at end of file
+	^ 'iVBORw0KGgoAAAANSUhEUgAAABgAAAAYCAYAAADgdz34AAAACXBIWXMAAA7DAAAOwwHHb6hk
```

Comment:
Why are some of the icons so much smaller in their serialized form? :)

Reply:
I downloaded the svg from the site and used a website to transfer them.  So I don't really know... If you tell me your method of conversion, I can try to convert them again

Reply:
conversion being svg to png

Reply:
I think I converted the icons to a higher resolution, something like 64x64. That might be the difference here. You also do not have to color them beforehand, you could change the color of the icon on the go via Form>>#dyed: (that would also allow for keeping the bw originals and also providing cached dyed versions :))

Reply:
Alrighty, I reversed the icon contents changes.
Somehow, calling `dyed:` on the icon forms results in this
![image](https://user-images.githubusercontent.com/33000454/147300796-5a40d077-08bd-45aa-80d0-f38784359d3e.png)

Do you know why this could be? 
Executed code
```(Icons 
		at: #babylonianProgrammingBarChart
		ifAbsentPut:[ Form fromBinaryStream: (Base64MimeConverter mimeDecodeToBytes: self barChartIconContents readStream) ])
		dyed: self iconColor.```

Reply:
The Form's bitmap (without `dyed:`) looks weird - it has values like 29829013 in it which may cause the block inside `dyed:` to only find transparent colors?